core-application/src/main/java/gohigher/application/service/ApplicationQueryService.java

```diff
@@ -0,0 +1,44 @@
+package gohigher.application.service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import gohigher.application.Application;
+import gohigher.application.port.in.ApplicationQueryPort;
+import gohigher.application.port.in.CalenderApplicationRequest;
+import gohigher.application.port.in.CalenderApplicationResponse;
+import gohigher.application.port.out.persistence.ApplicationPersistenceQueryPort;
+import gohigher.common.Process;
+import lombok.RequiredArgsConstructor;
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class ApplicationQueryService implements ApplicationQueryPort {
+
+	private final ApplicationPersistenceQueryPort applicationPersistenceQueryPort;
+
+	@Override
+	public List<CalenderApplicationResponse> findByMonth(CalenderApplicationRequest request) {
+		List<Application> applications = applicationPersistenceQueryPort.findByIdAndMonth(request.getUserId(),
+			request.getYear(), request.getMonth());
+
+		List<CalenderApplicationResponse> calenderApplicationResponses = new ArrayList<>();
+		for (Application application : applications) {
+			addScheduleResponses(application, calenderApplicationResponses);
+		}
+		return calenderApplicationResponses;
+	}
+
+	private void addScheduleResponses(Application application,
+		List<CalenderApplicationResponse> calenderApplicationRespons) {
```

Comment:
```suggestion
	private List<CalenderApplicationResponse> addScheduleResponses(Application application) {
```
이렇게 List를 반환하는 방법은 어떨까요? 자세히 기억이 안나는데 빈 배열을 파라미터로 넣어주고 외부에서 값을 채우는 방식의 단점이 있다고 들었습니다. 이유는 다시 한 번 찾아볼게요ㅋㅋ

Reply:
로직들을 stream으로 처리하도록 변경했습니다! 
별도로 이야기 나눈대로 map관련해서만 분리했습니다